lua/telescope/make_entry.lua

```diff
@@ -792,10 +792,11 @@ function make_entry.gen_from_registers(opts)
   end
 
   return function(entry)
+    local contents = vim.fn.getreg(entry)
     return make_entry.set_default_entry_mt({
       value = entry,
-      ordinal = entry,
-      content = vim.fn.getreg(entry),
+      ordinal = contents,
```

Comment:
Re: https://github.com/nvim-telescope/telescope.nvim/issues/996#issuecomment-1258966444

what we could do here is `string.format("%s %s", entry, contents)`

Try it and if it works as expected, feel free to open a new PR :)

Reply:
A while back I tried to solve this by changing the ordinal conditionally.

This is much better

Reply:
I tried to add that, and did a little git oopsy, and that branch doesn't exist anymore. I will add that later on. Also the pull request closed. Is that supposed to happen

Reply:
i already merged the PR

Reply:
you would need to open a new PR, like i said above.

So create a new branch from current master with the new changes.

I just saw the comment on the issue after merging

Reply:
My bad, didn't read the whole thing. I will just create a new PR.